packages/fhir-converter/src/templates/cda/Sections/Results.hbs

```diff
@@ -27,10 +27,8 @@
                         {{/with}}
                         {{#if obsEntry.observation.author.assignedAuthor}}
                             {{#with (evaluate 'Utils/GeneratePractitionerId.hbs' obj=obsEntry.observation.author.assignedAuthor) as |practitionerId|}}
-                                {{!--assert aaa toJsonString this.Id--}}
-                                {{#if practitionerId.Id}}
-                                    {{>References/Observation/performer.hbs ID=(generateUUID (toJsonString obsEntry.observation)) REF=(concat 'Practitioner/' practitionerId.Id )}},
-                                {{/if}}                            
+                                {{>Resources/Practitioner.hbs practitioner=obsEntry.observation.author.assignedAuthor ID=practitionerId.Id}},
```

Comment:
ok so I thought about it a bit more, and realized that the way Practitioners are generated in other places for top level resources in "inline" . For example, in notes.hbs:

```
{{#with (evaluate 'Utils/GeneratePractitionerId.hbs' obj=note.act.author.assignedAuthor) as |practitionerId|}}
                            {{>Resources/Practitioner.hbs practitioner=note.act.author.assignedAuthor ID=practitionerId.Id}},
                            {{>References/DiagnosticReport/performer.hbs ID=(generateUUID (toJsonString note.act)) REF=(concat 'Practitioner/' practitionerId.Id)}},
                        {{/with}}
```

Basically there are two ways to approach this problem of an ID being generated with no practitioner. Don't generate the ID (what I had before), or generate the Practitioner. I moved to the latter since thats the way the comparable handlebars templates went about things.

Reply:
This approach also means that while we might have some "junk" practitioners with no data in them, we now have 0% of ever missing a practitioner like what we were concerned about before. We can add a filter if we want to not generate a practitioner if there is no name (when there already is no id)